signup/signup.js

```diff
@@ -0,0 +1,152 @@
+import {
+  form,
+  email,
+  emailError,
+  passwordError,
+  eyeIcon,
+  TEST_EMAIL,
+  isEmail,
+  removeEmpty,
+  eyeIconOn,
+  eyeIconOff,
+} from '../src/common/sign.js';
+
+const passwordCheck = document.querySelector('.password_check');
+const passwordReCheck = document.querySelector('.password_recheck');
+
+const passwordErrorCheck = document.querySelector('.password-error-check');
+
+let isValid = [false, false, false];
```

Comment:
@JeongSabibi 

```suggestion
const isValid = [false, false, false];
```


원시타입 vs 참조타입에 대해 찾아보시면 좋을꺼 같아요.
참고 링크 드립니다.
- https://velog.io/@nomadhash/Java-Script-%EA%B9%8A%EC%9D%80-%EB%B3%B5%EC%82%AC%EC%99%80-%EC%96%95%EC%9D%80-%EB%B3%B5%EC%82%AC
- https://developer.mozilla.org/ko/docs/Glossary/Primitive
- https://hwani.dev/js-primitive-reference-types/

음...좀더 설명드리면 Array 는 객체입니다~!
- https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array

![image](https://github.com/codeit-bootcamp-frontend/4-Weekly-Mission/assets/56149367/48eea8f0-ce7e-43d3-93f6-3a389576a8a2)